report_qweb_signer/models/ir_actions_report.py

```diff
@@ -30,9 +30,15 @@ def _normalize_filepath(path):
 class IrActionsReport(models.Model):
     _inherit = "ir.actions.report"
 
+    def _check_valid_action_report(self):
+        if self.report_type != "qweb-pdf":
+            return True
```

Comment:
With this the behaviour is not the same as before ?

Also, I'd rename `_check_valid_action_report` to something more specific such as `_is_report_type_signable` ?

So 
```suggestion
    def _is_report_type_signable(self):
        return self.report_type == "qweb-pdf"
```

?